File: tests/week12/total_bill.js
function total_bill(bill_list) {
  let sum = 0;
  bill_list.forEach(element1 => {
    element1.forEach(element2 => {
      sum += parseFloat(element2.price.slice(1));
    });
  });
  return sum;
}

module.exports = total_bill;

File: tests/week12/total_bill.pl
#!/usr/bin/perl -w
$sum = 0;
while($line = <>) {
    my ($price) = $line =~ /\"price\": \"\$([\d\.]+)\"/g;
    if ($price) {
        $sum += $price;
    }
}
printf "\$%.2f\n", $sum;